src/derivative_wrappers.jl

```diff
@@ -40,55 +40,104 @@ function jacobian(f, x, integrator)
     alg = unwrap_alg(integrator, true)
     local tmp
     if alg_autodiff(alg)
-      J = jacobian_autodiff(f, x)
-      tmp = 1
+      if DiffEqBase.has_colorvec(integrator.f)
```

Comment:
I don't think we need to branch here. We can always take the colorvec one. Let's discuss this.

Reply:
I'm not sure that every f has colorvec.